SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -3239,6 +3239,10 @@ void USpatialNetDriver::TryFinishStartup()
 			}
 			else if (!StartupHandler->TryFinishStartup())
 			{
+				// We set bIsReadyToStart in TryFinishStartup a bit prematurely to get around ordering
+				// issues, setting bIsReadyToStart back to false shouldn't be needed but its here in case
+				// FSpatialServerStartupHandler::TryFinishStartup gets a new failure case.
+				bIsReadyToStart = false;
```

Comment:
I'd remove this and the `bIsReadyToStart = true` below and just let the StartupHandler manage this completely now

Reply:
Additionally I'd remove all the HACK references and just indicate that the StartupHandler is responsible for managing the NetDrivers ready state.